tests/integration/test_end_to_end_dcp.py

```diff
@@ -113,6 +113,9 @@ def test_smartseq2_run(self):
 
         self.check_manifest_contains_exactly_these_files(results_bundle_manifest, expected_files)
 
+        self.data_store.tombstone_bundle(runner.primary_bundle_uuids[0])
+        self.data_store.tombstone_bundle(runner.secondary_bundle_uuids[0])
+
 
```

Comment:
I think that this code should live at the end of DatasetRunner.run(), and should delete ALL primary and secondary bundles (by iterating through `primary_uuid_to_secondary_bundle_fqid_map`).   This way it will clear up after any test, and ones that generate more than one primary/secondary bundle.

Reply:
That seems sensible, fixed.